webapp/src/components/Chip/Chip.js

```diff
@@ -2,62 +2,59 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Icon } from 'semantic-ui-react'
 
-import { parcelType } from 'components/types'
-import './ParcelCoord.css'
+import './Chip.css'
 
-export default class ParcelCoord extends React.PureComponent {
+export default class Chip extends React.PureComponent {
   static propTypes = {
+    className: PropTypes.string,
     size: PropTypes.oneOf(['small', 'medium', 'large']),
-    parcel: parcelType,
     status: PropTypes.string,
+    children: PropTypes.node,
     onClick: PropTypes.func,
     onDelete: PropTypes.func
   }
 
   static defaultProps = {
+    className: '',
     size: 'medium',
     status: ''
   }
 
   handleOnClick = () => {
-    const { onClick, parcel } = this.props
-    if (onClick) {
-      onClick(parcel)
-    }
+    const { onClick } = this.props
+    if (onClick) onClick()
```

Comment:
I think that we are trying to always use brackets even though it has only one line